src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobListenerFactory.cs

```diff
@@ -123,58 +127,49 @@ internal class BlobListenerFactory : IListenerFactory
                 throw new ArgumentNullException("singletonManager");
             }
 
+            if (dispatchProcessorFactory == null)
+            {
+                throw new ArgumentNullException("dispatchProcessorFactory");
+            }
+
             _hostIdProvider = hostIdProvider;
-            _queueConfiguration = queueConfiguration;
+            // _queueConfiguration = queueConfiguration;
             _blobsConfiguration = blobsConfiguration;
             _exceptionHandler = exceptionHandler;
             _blobWrittenWatcherSetter = blobWrittenWatcherSetter;
-            _messageEnqueuedWatcherSetter = messageEnqueuedWatcherSetter;
+            // _messageEnqueuedWatcherSetter = messageEnqueuedWatcherSetter;
             _sharedContextProvider = sharedContextProvider;
-            _trace = trace;
-            _loggerFactory = loggerFactory;
+            // _trace = trace;
+            // _loggerFactory = loggerFactory;
             _functionId = functionId;
             _hostAccount = hostAccount;
             _dataAccount = dataAccount;
             _container = container;
             _input = input;
             _executor = executor;
             _singletonManager = singletonManager;
+            _dispatchProcessorFactory = dispatchProcessorFactory;
         }
 
         public async Task<IListener> CreateAsync(CancellationToken cancellationToken)
         {
             // Note that these clients are intentionally for the storage account rather than for the dashboard account.
             // We use the storage, not dashboard, account for the blob receipt container and blob trigger queues.
-            IStorageQueueClient primaryQueueClient = _hostAccount.CreateQueueClient();
-            IStorageBlobClient primaryBlobClient = _hostAccount.CreateBlobClient();
+
+            IStorageQueueClient primaryQueueClient = _hostAccount.CreateQueueClient(); // use for default poisonqueue
```

Comment:
nit: remove these new comments on the various clients. My comment above already indicates what they're used for